site/tests/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "commonjs",
+  "devDependencies": {
+    "@playwright/test": "1.23.3",
+    "playwright": "^1.23.3"
+  }
```

Comment:
What happens if we remove these?

Reply:
If it's just ESLint, we can silence it. Alternatively, we can see what's needed to remove `"type": "commonjs"` at all.

Reply:
It's just ESLint. Silenced it in https://github.com/blockprotocol/blockprotocol/pull/446/commits/b6eeea537ddc7374ace9d03daacf553b870d4ac9